neuralpp/test/quick_tests/symoblic/solver_compare_test.py

```diff
@@ -0,0 +1,61 @@
+"""
+Test cases comparing Z3 and SymPy solver.
+"""
+import pytest
+
+from sympy import symbols, Poly, PolynomialError, Interval, oo
+from sympy.solvers.inequalities import solve_rational_inequalities
+from z3 import *
+
+
+def solve_inequalities_sympy(ineq_list: list) -> None:
```

Comment:
The type of `ineq_list` would be typically typed as `List[Tuple[..., ..., ...]]` (not sure of the type for the SymPy data). You then need to import `List` and `Tuple` from package `typing` (PyCharm will suggest that).

Reply:
fixed.